Makefile

```diff
@@ -84,7 +84,6 @@ clean-gentype:
 
 clean:
 	dune clean
-	cargo clean --manifest-path rewatch/Cargo.toml && rm -f rewatch/rewatch
```

Comment:
I would prefer to keep this. `make clean` should clean everything including the rewatch build.

Reply:
then what's different with `make clean-all`?

It's inconvenient to rebuild `rewatch` every time I run `make clean test`. we can add `make clean-rewatch`

Reply:
Ok, this is a good question. 😄 This Makefile has grown organically.

I guess it's also fine to have `clean-rewatch` and put that into `clean-all`, too.